openid-data-classes/src/commonMain/kotlin/at/asitplus/openid/KeyAttestationRequired.kt

```diff
@@ -0,0 +1,19 @@
+package at.asitplus.openid
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class KeyAttestationRequired(
+    /**
+     * OID4VCI: OPTIONAL. Array defining values specified in Appendix D.2 accepted by the Credential Issuer.
+     */
+    @SerialName("key_storage")
+    val keyStorage: Collection<String>? = null,
```

Comment:
Why not make these values an enum?

Reply:
Because I assume that those values will change, and because they have no impact as of yet